pydra/engine/audit.py

```diff
@@ -218,13 +224,16 @@ def audit_task(self, task):
         }
         entity_id = f"uid:{gen_uuid()}"
         entity_message = {
```

Comment:
you should create entity message for each file from the input

Reply:
Do you recommend putting everything into a function and then calling it for each input?

Reply:
Disregard the last message, I believe I fixed it.